lib/Backend/LowerMDShared.cpp

```diff
@@ -8683,24 +8687,21 @@ void LowererMD::GenerateFastInlineBuiltInCall(IR::Instr* instr, IR::JnHelperMeth
             //     if (shouldCheckNegZero) {
             //         CMP roundedFloat, 0
             //         JA $setZero
-            //         JE $negZeroTest
-            //         J $bailout
+            //         JP $skipRoundSd (NaN)
```

Comment:
```roundedFloat``` should not be NaN here, because the NaN case has been handled above by
```
CMP 0.5, roundedFloat
JA $ltHalf
CMP TwoToFraction, roundedFloat
JA $addHalfToRoundSrcLabel
J $skipRoundSd (NaN is already handled here)
```